code/game/turfs/open/floor/fancy_floor.dm

```diff
@@ -11,13 +11,15 @@
 	desc = "Stylish dark wood."
 	icon_state = "wood"
 	floor_tile = /obj/item/stack/tile/wood
-	broken_states = list("wood-broken", "wood-broken2", "wood-broken3", "wood-broken4", "wood-broken5", "wood-broken6", "wood-broken7")
 	footstep = FOOTSTEP_WOOD
 	barefootstep = FOOTSTEP_WOOD_BAREFOOT
 	clawfootstep = FOOTSTEP_WOOD_CLAW
 	heavyfootstep = FOOTSTEP_GENERIC_HEAVY
 	tiled_dirt = FALSE
 
+/turf/open/floor/wood/setup_broken_states()
+	broken_states = list("wood-broken", "wood-broken2", "wood-broken3", "wood-broken4", "wood-broken5", "wood-broken6", "wood-broken7")
```

Comment:
Either make them all stringlists on each proc redefinition, or make something like this on Initialize:

```
if (broken_states)
	stack_trace("the message I showed you before on discord")
else
	var/list/new_broken_states = setup_broken_states()
	if(new_broken_states)
		broken_states = string_list(new_broken_states)
```